magefile.go

```diff
@@ -1,17 +1,59 @@
-// +build mage
+//go:build mage
 
 package main
 
 import (
-	// mage:import
-	"get.porter.sh/porter/mage/releases"
+	"get.porter.sh/magefiles/mixins"
 )
 
-// We are migrating to mage, but for now keep using make as the main build script interface.
+const (
+	mixinName    = "arm"
+	mixinPackage = "get.porter.sh/mixin/arm"
+	mixinBin     = "bin/mixins/" + mixinName
+)
+
+var magefile = mixins.NewMagefile(mixinPackage, mixinName, mixinBin)
```

Comment:
Thanks for updating the magefile! Now that we have it, the next step is to remove the Makefile, and then update the azure pipeline file to use mage instead of make.